librad/src/git/include.rs

```diff
@@ -92,6 +95,12 @@ impl<Path> Include<Path> {
         }
     }
 
+    pub fn add_remote(&mut self, url: LocalUrl, peer: PeerId, handle: &str) -> Result<(), Error> {
```

Comment:
If the `handle` is rejected as a refname, we would fail the entire `from_tracked_users`, right? 

Perhaps we should actually be more restrictive about what can be used as a name, so we can define this as `impl Into<RefLike>`... :thinking:

Reply:
Ya, I had a similar thought. Although the name is actually `<handle>@<peer>`. If `handle` is `RefLike` would that imply `<handle>@<peer>` is `RefLike`? Probably yes, right? It'd be `handle.into().join("@").join(peer)`.

I'd prefer create a follow up ticket for making the remote name `RefLike`, if that's ok with you? :)

Reply:
Created an issue here: https://github.com/radicle-dev/radicle-link/issues/414

Reply:
> handle.into().join("@").join(peer)

Except as per commandment 9 of [git-check-ref-format](https://git-scm.com/docs/git-check-ref-format), the single character "@" is not a valid refname. :low_brightness:

Reply:
That's #414's problem :grimacing: